src/components/common/interfaces.ts

```diff
@@ -0,0 +1,5 @@
+export type CopyDisabledProps = {
+  onCopy?: (event: React.ClipboardEvent<HTMLElement>) => void;
```

Comment:
There is already a type existing, called InputBaseProps, could you refactor it with yours please :)

Reply:
Thank you @AliciaSymphony for your comment. I updated the code. :)